Shopify/shopify-cli
# Extension serve port selection

### WHY are these changes introduced?

Windows compatibility

### WHAT is this pull request doing?

- Warn if development server binary was not found
- extension serve port selection

### How to test your changes?

- Create an extension configuration using `shopify extension create`. Answer the questions to complete the configuration.
- Run `ngrok http <port_number>`
- `cd` into the extension directory and run `shopify extension serve --port=<port_number>`
- Respond to the prompts and ensure the server starts up and the resulting links are active.
- Test that curl `http://localhost:<port_number>/extensions` returns a valid JSON response



### Post-release steps

None



### Update checklist
